WooCommerce/Classes/ViewRelated/Dashboard/MyStore/NewOrdersViewController.swift

```diff
@@ -60,6 +60,22 @@ extension NewOrdersViewController {
 
         StoresManager.shared.dispatch(action)
     }
+
+    func syncOrderStatus(onCompletion: ((Error?) -> Void)? = nil) {
+        guard let siteID = StoresManager.shared.sessionManager.defaultStoreID else {
+            onCompletion?(nil)
+            return
+        }
+
+        let action = OrderStatusAction.retrieveOrderStatuses(siteID: siteID) { (orderStatuses, error) in
```

Comment:
@mindgraffiti This is great place to call this. Any thoughts on also [dispatching the same action in `StoresManager`](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/Yosemite/StoresManager.swift#L228)? 

I am wondering if we should fetch these values anytime we change stores?

Reply:
yesss thanks! Added to `StoresManager`

Reply:
(per Slack discussion, I will not fetch values in the StorePickerVC b/c it could potentially cause merge conflicts atm)